zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -9,10 +9,12 @@
 
 import static io.camunda.zeebe.protocol.record.ValueType.AUTHORIZATION;
 import static io.camunda.zeebe.protocol.record.ValueType.DECISION;
+
```

Comment:
❌ remove this empty line or run spotless to pass the failing java checks

Reply:
Yes, already did! Thank you